src/OpenTelemetry.Exporter.Prometheus.Shared/PrometheusExporterOptions.cs

```diff
@@ -15,61 +15,25 @@
 // </copyright>
 
 using System;
-using System.Collections.Generic;
 using OpenTelemetry.Internal;
 
-namespace OpenTelemetry.Exporter
+#if PROMETHEUS_ASPNETCORE
+namespace OpenTelemetry.Exporter.Prometheus.AspNetCore
+#elif PROMETHEUS_HTTPLISTENER
+namespace OpenTelemetry.Exporter.Prometheus.HttpListener
+#else
```

Comment:
This `#else` check is not required.

Reply:
If we remove it I don't think the shared project will build by itself.